packages/crypto/test/pbkdf2.node.test.ts

```diff
@@ -0,0 +1,38 @@
+import { bufferFromString, pbkdf2 } from '..';
+
+/**
+ * @group node
```

Comment:
```suggestion
 * @group node
 * @group browser
```

Reply:
These tests are intended to run specifically on Node as they relate to a method that is useful for multi-threaded node environments which is not applicable in the browser.